mashup-example/using-build-tool/qlik-sense-saas/main.js

```diff
@@ -5,9 +5,9 @@ import './style.css';
 
 async function run() {
   const app = await connect({
-    url: '<Qlik tenant url>', // 'xxxx.us.qlik.com',
-    webIntegrationId: '<Qlik web integration id>', // 'xxx-xxxxxxx-xxxxxxxx',
-    appId: '<App id>', // 'xxxx-xxx-xxx-xxx-xxxxxxx',
+    url: 'wastelands.us.qlikcloud.com', // 'xxxx.us.qlik.com',
+    webIntegrationId: 'TGzZjWAv7OK6F4lp0oQhf_7Y-XCK_DtN', // 'xxx-xxxxxxx-xxxxxxxx',
+    appId: '95ab782b-1d7d-4ecc-acaa-4dd17a3841da', // 'xxxx-xxx-xxx-xxx-xxxxxxx',
```

Comment:
I do not think it is good to share those to the public

Reply:
That's the mashup example.

Reply:
removed

Reply:
looks like they are still here